genesyscloud/team/resource_genesyscloud_team_schema.go

```diff
@@ -0,0 +1,92 @@
+package team
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	gcloud "terraform-provider-genesyscloud/genesyscloud"
+	resourceExporter "terraform-provider-genesyscloud/genesyscloud/resource_exporter"
+	registrar "terraform-provider-genesyscloud/genesyscloud/resource_register"
+)
+
+/*
+resource_genesycloud_team_schema.go holds four functions within it:
+
+1.  The registration code that registers the Datasource, Resource and Exporter for the package.
+2.  The resource schema definitions for the team resource.
+3.  The datasource schema definitions for the team datasource.
+4.  The resource exporter configuration for the team exporter.
+*/
+const resourceName = "genesyscloud_team"
+
+// SetRegistrar registers all of the resources, datasources and exporters in the package
+func SetRegistrar(regInstance registrar.Registrar) {
```

Comment:
When this is ready to be released to production we need to make sure that this method is called from the main.go.   Right now I was looking at the code and without you registering this in the main.go, you will run into a situation where the code looks like its all there, but you will get an error when you try to run this.